src/main/java/Barney.java

```diff
@@ -1,8 +1,21 @@
 public class Barney {
+    private static String LONG_LINE = "_________________________________________________";
+
     public static void main(String[] args) {
-        String welcomeText = "Hello, I am Barney <RAWR>, what can I do for you?";
-        System.out.println(welcomeText);
-        String endingText = "Goodbye, I am Barney <RAWR>, see you next time!";
-        System.out.println(endingText);
+        welcomeText();
+        endingText();
     }
+
+    public static void welcomeText() {
+        String text = "Hello, I am Barney <RAWR>, what can I do for you?";
+        System.out.println(text);
+        System.out.println(LONG_LINE);
+    }
+
+    public static void endingText() {
+        String text = "Goodbye, I am Barney <RAWR>, see you next time!";
+        System.out.println(text);
+        System.out.println(LONG_LINE);
+    }
```

Comment:
I really like how you refactored the code into reusable functions right from the start! It really demonstrated how you kept OOP design principle in mind, especially at this level the instructions hadn't asked for refactoring yet:)

Reply:
Thank you so much!